git_pw/patch.py

```diff
@@ -91,7 +91,10 @@ def _format_series(series):
         prefix = ''
 
     # TODO(stephenfin): We might want to make this machine readable?
-    click.echo(tabulate(output, ['Property', 'Value'], tablefmt='psql'))
+    if echo:
+        click.echo(tabulate(output, ['Property', 'Value'], tablefmt='psql'))
```

Comment:
To be honest, I never intended for this to be used as a library. At present, we don't have a stable API so I don't think it's a good idea to be relying on this. Instead, we should probably separate things into something like `api` and `cli` directories. This is a sizable rework but I might be able to tackle it this weekend.

Reply:
I understand.

Even with all the patchwork schizma and unstable API, we need this internally. 

For now, I'll close this pull request and use git-pw with couple of these hacks.

Let me know, once you have the split structure thought-out, I may be able to help with rework.

Thank you.

Reply:
No problem. Thanks for understanding :smile: Let's see how my weekend goes